connect/src/routes/cctp/manual.ts

```diff
@@ -0,0 +1,130 @@
+import { Signer, CircleTransferDetails, TransactionId } from "@wormhole-foundation/sdk-definitions";
+import { ManualRoute, TransferParams, ValidatedTransferParams, ValidationResult } from "../route";
+import { TransferReceipt, TransferState, isAttested } from "../../wormholeTransfer";
+import { CircleTransfer, CircleTransferProtocol } from "../../protocols/cctpTransfer";
+import { signSendWait } from "../../common";
+import { Network, circle, normalizeAmount, Chain } from "@wormhole-foundation/sdk-base";
+
+export namespace CCTPRoute {
+  export type Options = {
+    payload?: Uint8Array;
+  };
+
+  export type NormalizedParams = {
+    amount: bigint;
+  };
+
+  export interface ValidatedParams extends ValidatedTransferParams<Options> {
+    normalizedParams: NormalizedParams;
+  }
+}
+
+type Op = CCTPRoute.Options;
+type Vp = CCTPRoute.ValidatedParams;
+type Tp = TransferParams<Op>;
+type Vr = ValidationResult<Op>;
+
+export class CCTPRoute<N extends Network> extends ManualRoute<N, Op> {
+  static getDefaultOptions(): Op {
+    return {
+      payload: undefined,
+    };
+  }
+
+  async isSupported(): Promise<boolean> {
```

Comment:
This should be checking if the chain context returns true for `supportsCircleBridge`